generators/server/templates/src/main/java/_package_/service/dto/UserDTO.java.ejs

```diff
@@ -104,6 +104,11 @@ public class <%= user.dtoClass %> implements Serializable {
             Objects.equals(getLogin(), userDTO.getLogin());
     }
 
+    @Override
+    public int hashCode() {
+        return Objects.hashCode(getId());
```

Comment:
Id only is used for entities, I think DTO should be calculated using fields too.

Reply:
@mshima I updated